scripts/install_dev_python_modules.py

```diff
@@ -48,76 +48,138 @@ def main(
 
     # Supported on all Python versions.
     install_targets += [
-        "-e python_modules/dagster[pyright,ruff,test]",
-        "-e python_modules/dagster-pipes",
-        "-e python_modules/dagster-graphql",
-        "-e python_modules/dagster-test",
-        "-e python_modules/dagster-webserver",
-        "-e python_modules/dagit",
-        "-e python_modules/automation",
-        "-e python_modules/libraries/dagster-managed-elements",
-        "-e python_modules/libraries/dagster-airbyte",
-        "-e python_modules/libraries/dagster-aws[test]",
-        "-e python_modules/libraries/dagster-celery",
-        "-e python_modules/libraries/dagster-celery-docker",
-        "-e python_modules/libraries/dagster-dask[yarn,pbs,kube]",
-        "-e python_modules/libraries/dagster-databricks",
-        "-e python_modules/libraries/dagster-datadog",
-        "-e python_modules/libraries/dagster-datahub",
-        "-e python_modules/libraries/dagster-dbt",
-        "-e python_modules/libraries/dagster-docker",
-        "-e python_modules/libraries/dagster-gcp",
-        "-e python_modules/libraries/dagster-gcp-pandas",
-        "-e python_modules/libraries/dagster-gcp-pyspark",
-        "-e python_modules/libraries/dagster-embedded-elt",
-        "-e python_modules/libraries/dagster-fivetran",
-        "-e python_modules/libraries/dagster-k8s",
-        "-e python_modules/libraries/dagster-celery-k8s",
-        "-e python_modules/libraries/dagster-github",
-        "-e python_modules/libraries/dagster-mlflow",
-        "-e python_modules/libraries/dagster-mysql",
-        "-e python_modules/libraries/dagster-looker",
-        "-e python_modules/libraries/dagster-openai",
-        "-e python_modules/libraries/dagster-pagerduty",
-        "-e python_modules/libraries/dagster-pandas",
-        "-e python_modules/libraries/dagster-papertrail",
-        "-e python_modules/libraries/dagster-postgres",
-        "-e python_modules/libraries/dagster-prometheus",
-        "-e python_modules/libraries/dagster-pyspark",
-        "-e python_modules/libraries/dagster-shell",
-        "-e python_modules/libraries/dagster-slack",
-        "-e python_modules/libraries/dagster-spark",
-        "-e python_modules/libraries/dagster-ssh",
-        "-e python_modules/libraries/dagster-twilio",
-        "-e python_modules/libraries/dagstermill",
-        "-e integration_tests/python_modules/dagster-k8s-test-infra",
-        "-e python_modules/libraries/dagster-azure",
-        "-e python_modules/libraries/dagster-msteams",
-        "-e python_modules/libraries/dagster-deltalake",
-        "-e python_modules/libraries/dagster-deltalake-pandas",
-        "-e python_modules/libraries/dagster-deltalake-polars",
-        "-e helm/dagster/schema[test]",
-        "-e .buildkite/dagster-buildkite",
-        "-e examples/experimental/dagster-blueprints",
-        "-e examples/experimental/dagster-airlift",
+        "-e",
+        "python_modules/dagster[pyright,ruff,test]",
```

Comment:
might be nice to refactor this to programmatically add the `-e` s against a list of just the paths (ie `editable_target_paths`) to make the final `cli_args`

Reply:
Something like this? https://github.com/dagster-io/dagster/pull/23580/commits/db4180abfa9161f5e503e78cc234bdfa26f1f24f